### ISSUE TITLE
two tableview problem

### ISSUE THREAD

**Issue by koust** [adamshin/SwiftReorder#71]
Hi,

I have two tableviews in the vc. I can move from row 1 to row 0 in my first tableview but if I want to move from row 1 to row 0 in my second tableview, it doesn't work. 

Can u help me about this problem?

**Comment by adamshin** [comment]
This library only allows dragging cells within a single tableview. If you need to drag cells between multiple table views, you should use the new iOS drag-and-drop apis.

**Comment by koust** [comment]
Actually, I asked a different question. I don't wanna use to drag cells between multiple table views. I use as a single each tableviews. Only I have two table views. I drag from row (1,2,3,4,5) to row 0 on my first tableview and it's working. but the second tableview is not working if I drag from (1,2,3,4,5) to row 0.  

I attached a video about the problem.


https://youtu.be/cs_FxzoW-Io

**Comment by adamshin** [comment]
I see, I misread your question. Do you have the same view controller as the reorder delegate for both tableviews? It looks like that might be the issue.

**Comment by koust** [comment]
I added reorder delegate for both tableviews on the same view controller. I mean yes

**Comment by adamshin** [comment]
In your `TableViewReorderDelegate` methods, are you checking which tableview is being interacted with before updating your data model? Like this:

```swift
extension MyViewController: TableViewReorderDelegate {
    func tableView(_ tableView: UITableView,
        reorderRowAt sourceIndexPath: IndexPath,
        to destinationIndexPath: IndexPath) {
        
        if tableView == firstTableView {
            // Update first data model
        } else if tableView == secondTableView {
            // Update second data model
        }
    }
}
```

**Comment by koust** [comment]
Exactly, like this. If drag cells from row (1,2,3,...) to 0, this fuction doesnt work.
but it work  from row 0 to 1 or row 2 to 3 etc.
